reinforcement_learning/Lesson-3-Value-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>Minor typos. Corrected: "How can <em>Monte Carlo Method</em> be applied..." and "...think of a game that you have played before"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>